src/main.js

```diff
@@ -61,9 +61,13 @@ new Vue({
   apolloProvider,
   vuetify,
   beforeCreate() {
-    this.$intercom.boot({
-      user_id: uuidv4()
-    });
+    const userId = this.$route.query.intercomid
+      ? this.$route.query.intercomid
+      : uuidv4();
+    !this.$route.query.intercomid
```

Comment:
This is unnecessary as you already checked it the line before. Just call boot with userId.

Reply:
fixed